src/index.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import { Provider } from 'react-redux';
-import { createStore } from 'redux';
+// import { createStore } from 'redux';
 import initialState from './default/index';
-import books from './reducers/books';
+import { createBook } from './actions/index';
+// import books from './reducers/books';
 import App from './components/App';
+import store from './reducers/index';
 
-const store = createStore(books, initialState);
+// const store = createStore(books, initialState);
+
+initialState.map((book) => store.dispatch(createBook(book)));
```

Comment:
Oops 🤦 , Let's try to stick to best practice, you made this correction in milestone 2 to my surprise you changed it back, are you having any difficulty doing it the right way?

Kindly update this, 👍 


![image](https://user-images.githubusercontent.com/31897434/128015359-ecf9ce77-7b03-48a9-a121-56185fc51b40.png)

Reply:
Please I seem not to be able to get around this. Your help will be much appreciated.

Reply:
@happiguru So sorry I'm just seeing this, I was out.

I hope it is all good now.

Reply:
Yes it is okay now. Thank you!!